.gitignore

```diff
@@ -11,3 +11,4 @@ test/services/tokens
 # IDE
 /*.iml
 .idea
+/.vscode
```

Comment:
This is a kind of Pull Request pollution as it does not serves the purpose of the current intent.
I know it is nothing (1 line of code) but it should not be here in my opinion, and should have it's dedicated Pull Request with its own commit message that describes a new intent.

Reply:
Fixed by splitting this PR into 2 : #100 and #99